WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -99,7 +100,7 @@ private static void save(String table, Person person, SQLiteDatabase database) {
         switch (table) {
             case TEAM_TABLE:
                 values.put("user_name", person.getUsername());
-                values.put("role", person.getRole());
+                values.put("role", person.getRole().toString());
```

Comment:
I think we need to make sure `getRole()` isn't null here, otherwise there could be a NPE.